Obsidian.API/_Types/ChatMessage.cs

```diff
@@ -41,11 +44,91 @@ public IEnumerable<ChatMessage> GetExtras()
+    /// </summary>
+    /// <param name="a">The left chat message on which the right one gets appended</param>
+    /// <param name="b">The right chat message which will be appended</param>
+    /// <returns>The current chat message. Equally to the left chat message</returns>
+    public static ChatMessage operator +(ChatMessage a, ChatMessage b) => a.AddExtra(b);
+
+    /// <summary>
+    /// Adds the given <see cref="ChatColor"/> to the text of the given <see cref="ChatMessage"/>.
+    /// </summary>
+    /// <param name="a">The message on which the chat color gets appended</param>
+    /// <param name="b">The chat color which will be appended</param>
+    /// <returns>The current chat message. Equally to the given chat message</returns>
+    public static ChatMessage operator +(ChatMessage a, ChatColor b) => a.AppendColor(b);
+
+    /// <summary>
+    /// Creates a new <see cref="ChatMessage"/> object with plain text.
+    /// </summary>
+    /// <param name="text">The text of the <see cref="ChatMessage"/></param>
+    /// <param name="reformat">Whether to reformat the text via <see cref="ReformatFormattingPrefixes"/> or not</param>
+    /// <returns>The created <see cref="ChatMessage"/> object</returns>
+    public static ChatMessage Simple(string text, bool reformat = true) => new() { Text = reformat ? ReformatFormattingPrefixes(text) : text };
+    
     /// <summary>
     /// Creates a new <see cref="ChatMessage"/> object with plain text.
     /// </summary>
-    public static ChatMessage Simple(string text) => new() { Text = text.Replace('&', '§') };
-    public static ChatMessage Simple(string text, ChatColor color) => new() { Text = $"{color}{text}" };
+    /// <param name="text">The text of the <see cref="ChatMessage"/></param>
+    /// <param name="color">The <see cref="ChatColor"/> of the <see cref="ChatMessage"/></param>
+    /// <param name="reformat">Whether to reformat the text via <see cref="ReformatFormattingPrefixes"/> or not</param>
+    /// <returns>The created <see cref="ChatMessage"/> object</returns>
+    public static ChatMessage Simple(string text, ChatColor color, bool reformat = true) => new()
+    {
+        Text = $"{color}{(reformat ? ReformatFormattingPrefixes(text) : text)}"
+    };
+
+    /// <summary>
+    /// Appends an underlying <see cref="ClickEvent"/> (<see cref="ClickComponent"/>) to the given <see cref="ChatMessage"/>.
+    /// </summary>
+    /// <param name="message">The message which will hold the <see cref="ClickComponent"/></param>
+    /// <param name="action">The action which will be executed when clicking</param>
+    /// <param name="value">The value which will be executed with the action</param>
+    /// <param name="translate">The translate value</param>
+    /// <returns>The given <see cref="ChatMessage"/></returns>
+    public static ChatMessage Click(ChatMessage message, EClickAction action, string value, string translate = "")
```

Comment:
I'm fine with this, but I was thinking that having this be an instance method called `WithClick` would be more... conventional? What do you think @Naamloos / @Tides ?

Reply:
I think this is fine. Would work great with the example given in the original PR message :)

Reply:
it implemented